APMrover2/sensors.cpp

```diff
@@ -83,7 +83,7 @@ void Rover::read_sonars(void)
                 obstacle.detected_count++;
             }
             if (obstacle.detected_count == g.sonar_debounce) {
-                gcs_send_text_fmt("Sonar obstacle %u cm",
+                gcs_send_text_fmt(MAV_SEVERITY_NOTICE, "Sonar obstacle %u cm",
```

Comment:
This can be MAV_SEVERITY_INFO.  The only difference is this is using one sonar whereas the above code uses 2.